src/main/java/com/lambdatest/jenkins/freestyle/MagicPlugDescriptor.java

```diff
@@ -118,6 +123,78 @@ public ListBoxModel doFillResolutionItems(@QueryParameter String operatingSystem
 		return items;
 	}
 
+	public ListBoxModel doFillPlatformNameItems() {
+		Map<String, String> supportedPlatforms = AppAutomationCapabilityService.getPlatformNames();
+		logger.info("OS triggered : " + supportedPlatforms);
+		ListBoxModel items = new ListBoxModel();
+		items.add(Constant.DEFAULT_PLATFORM_NAME_VALUE, Constant.EMPTY);
+		supportedPlatforms.forEach((key, value) -> {
+			items.add(value, key);
+		});
+		return items;
+	}
+
+	public ListBoxModel doFillBrandNameItems(@QueryParameter String platformName) {
+		ListBoxModel items = new ListBoxModel();
+		if (StringUtils.isBlank(platformName)) {
+			items.add(Constant.DEFAULT_BRAND_NAME_VALUE, Constant.EMPTY);
+			return items;
+		}
+		Set<String> supportedBrands = AppAutomationCapabilityService.getBrandNames(platformName);
+		logger.info("Brand Names triggered : " + supportedBrands);
+		if (!CollectionUtils.isEmpty(supportedBrands)) {
+			supportedBrands.forEach(br -> {
+				items.add(br, br);
+			});
+		}
+		return items;
+	}
+	
+	public ListBoxModel doFillDeviceNameItems(@QueryParameter String platformName, @QueryParameter String brandName) {
+		ListBoxModel items = new ListBoxModel();
+		if (!StringUtils.isBlank(platformName) && StringUtils.isBlank(brandName)) {
+			brandName = "Asus";
+			System.out.println("Asus added");
+		} else if (StringUtils.isBlank(platformName) || StringUtils.isBlank(brandName)) {
+			items.add(Constant.DEFAULT_DEVICE_NAME_VALUE, Constant.EMPTY);
+			return items;
+		}
+		Set<String> supportedDevices = AppAutomationCapabilityService.getDeviceNames(platformName, brandName);
+		logger.info("Device Names triggered : " + supportedDevices);
+		if (!CollectionUtils.isEmpty(supportedDevices)) {
+			supportedDevices.forEach(br -> {
+				items.add(br, br);
+			});
+		}
+		return items;
+	}
+
+	public ListBoxModel doFillDeviceVersionItems(@QueryParameter String platformName,
+			@QueryParameter String deviceName) {
+		ListBoxModel items = new ListBoxModel();
+		if (!StringUtils.isBlank(platformName) && StringUtils.isBlank(deviceName)) {
+			deviceName = "Zenfone 6";
+			System.out.println("Zenfone 6 added");
```

Comment:
@rohithmone-lt 
**Please scan all `System.out.println` statements in project and remove them**

Reply:
Should I use logger.info to print these statements instead of System.out.println or shall I completely remove them?

Reply:
Done